package-lock.json

```diff
@@ -3577,6 +3577,7 @@
     },
     "node_modules/npm/node_modules/lodash._baseindexof": {
       "version": "3.1.0",
+      "extraneous": true,
```

Comment:
The changes in package-lock should not be part of this PR!?

Reply:
I thought we should push such changes no? I can ofc remove it but the lock file should be up to date in general

Reply:
Mhh, not sure why it is outdated in the first place. Either way is fine for me.